css-four-column-layout/styles.css

```diff
@@ -0,0 +1,105 @@
+* {
+  font-family: Barlow, sans-serif;
+}
+
+header {
+  text-align: center;
+  font-weight: 900;
+  width: auto;
+}
+
+h3 {
+  text-align: center;
+  font-weight: 900;
+  font-size: 20px;
+}
+
+p {
+  text-align: center;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  width: auto;
+  align-content: center;
+}
+
+.column-one-fourth {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: space-evenly;
+  width: calc(100% / 4);
+  height: auto;
+}
+
+.phone-app {
+  background-image: url("images/screen-1.png");
+  background-size: 100%;
+  width: 300px;
+  height: 460px;
+  margin-bottom: 30px;
+}
+
+.phone-trip {
+  background-image: url("images/screen-2.png");
+  background-size: 100%;
+  width: 300px;
+  height: 460px;
+  margin-bottom: 30px;
+}
+
+.phone-service {
+  background-image: url("images/screen-3.png");
+  background-size: 100%;
+  width: 300px;
+  height: 460px;
+  margin-bottom: 30px;
+}
+
+.phone-mileage {
+  background-image: url("images/screen-4.png");
+  background-size: 100%;
+  width: 300px;
+  height: 460px;
+  margin-bottom: 30px;
+}
```

Comment:
since all of there classes are doing the same thing except for the background images, I would create one class with all the css except for the image, and then add that class to each element, then create a class with just the background image info for each image and add that class to the set the image. It will reduce redundant code.